src/components/animated/_liquid.less

```diff
@@ -38,11 +45,11 @@
     .bc(@color);
 
     &:hover::before {
```

Comment:
you need to add `, &.animated:before` as well to work for the animated class. Same for the rule for `:after`